int32 queryIdx
int32 trainIdx
int32 imgIdx
float32 distance